contrib/actionchain/usecases.py

```diff
@@ -41,7 +41,7 @@ def findServerByHostname(self, fullHostName):
 
 class ScenarioRunner(SMConnect):
     """
-    Admin scenario for the SUSE Manager, utilizing Action Chains.
+    Admin scenario for the SUSE Multi-Linux Manager, utilizing Action Chains.
```

Comment:
As discussed via chat,  the whole contrib folder seems to just be legacy code. 

```suggestion
    Admin scenario for the SUSE Manager, utilizing Action Chains.
```